docs/source/AdministratorGuide/Tutorials/basicTutoSetup.rst

```diff
@@ -0,0 +1,731 @@
+  2019-04-11 14:46:41 UTC dirac-install [NOTICE]  Installing DIRAC:v6r21
+  2019-04-11 14:46:41 UTC dirac-install [NOTICE]  Retrieving http://diracproject.web.cern.ch/diracproject/tars/DIRAC-v6r21.tar.gz
+  2019-04-11 14:46:41 UTC dirac-install [NOTICE]  Retrieving http://diracproject.web.cern.ch/diracproject/tars/DIRAC-v6r21.md5
+  2019-04-11 14:46:42 UTC dirac-install [NOTICE]  Deploying scripts...
+  Scripts will be deployed at /home/diracuser/DIRAC/scripts
+  Inspecting DIRAC module
+  2019-04-11 14:46:42 UTC dirac-install [NOTICE]  Installing DIRAC OS ...
+  2019-04-11 14:46:42 UTC dirac-install [NOTICE]  Retrieving https://diracos.web.cern.ch/diracos/releases/diracos-1.0.0.tar.gz
+  .........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................2019-04-11 14:46:46 UTC dirac-install [NOTICE]  Retrieving https://diracos.web.cern.ch/diracos/releases/diracos-1.0.0.md5
+  2019-04-11 14:47:02 UTC dirac-install [NOTICE]  Fixing externals paths...
+  2019-04-11 14:47:02 UTC dirac-install [NOTICE]  Running externals post install...
+  2019-04-11 14:47:02 UTC dirac-install [NOTICE]  Creating /home/diracuser/DIRAC/bashrc
+  2019-04-11 14:47:02 UTC dirac-install [NOTICE]  Defaults written to defaults-DIRAC.cfg
+  2019-04-11 14:47:02 UTC dirac-install [NOTICE]  Executing /home/diracuser/DIRAC/scripts/dirac-externals-requirements...
+  2019-04-11 14:47:03 UTC dirac-install [NOTICE]  DIRAC properly installed
+
+You will notice that among other things, the installation created a ``~/DiracInstallation/bashrc`` file. This file must be sourced whenever you want to use dirac client.
+
+In principle, your system administrator will have managed the CA for you. In this specific case, since we have our own CA, we will just link the client installation CA with the server one::
+
+  mkdir -p ~/DiracInstallation/etc/grid-security/
+  ln -s /opt/dirac/etc/grid-security/certificates/ ~/DiracInstallation/etc/grid-security/certificates
+
+The last step is to configure the client to talk to the proper configuration service. This is easily done by creating a ``~/DiracInstallation/etc/dirac.cfg`` file with the following content::
```

Comment:
For this specific case the net result is the same, but normally the configuration is done by calling the `dirac-configure` script: https://dirac.readthedocs.io/en/latest/UserGuide/GettingStarted/InstallingClient/index.html#configuring-client

Reply:
Ah ? for my education, what does it do exactly ? It seems to replace `LocalInstallation` with `DIRAC` no ?

Reply:
`dirac-configure` at a minimum will fill correctly the `dirac.cfg`, but a few more things will or can be done, like take care of the CAs. Try it out.